Formula/docker-ls.rb

```diff
@@ -0,0 +1,26 @@
+class DockerLs < Formula
+  desc "Tools for browsing and manipulating docker registries."
+  homepage "https://github.com/mayflower/docker-ls"
+  url "https://github.com/mayflower/docker-ls.git",
+    :tag => "v0.3.1",
+    :revision => "d80310976c9707e261e57ebfa9acf4e0b1781460"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    (buildpath/"src/github.com/mayflower/docker-ls").install buildpath.children
+
+    system "go", "generate", "github.com/mayflower/docker-ls/lib"
+    system "go", "install", "github.com/mayflower/docker-ls/cli/..."
+
+    bin.install buildpath/"bin/docker-ls"
+    bin.install buildpath/"bin/docker-rm"
+  end
+
+  test do
+    system "#{bin}/docker-ls", "version"
```

Comment:
This needs to be a test that exercises and verifies the functionality of the software.

Reply:
Please see my comment above - any more comprehensive test requires a connection to a docker registry and would introduce spurious failures if the connection is unstable or the registry is down.

Reply:
You could also check for expected failure case.

Reply:
A non-trivial test (i.e., not --help, not --version) is a requirement for all new formulae.

Reply:
Alright, I'll add a test against the official docker hub. Might be a few days until I find time, though.

Reply:
I've added two additional nontrivial tests that access the docker hub. Is there anything else I can do?